<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:rgb(54,54,54);
color:white;
font:13px;
}
QGroupBox{
border:1px solid grey;
border-radius:5px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:3px;
border:1px solid grey;
}
QPushButton:hover{
border:2px solid white;
color:rgb(30,30,30);
}
QPushButton:pressed{
border:3px solid orange;
}
QPushButton#pushButton_credits_new_creditee{
background-color:green;
}
QLineEdit{
border:1px solid grey;
border-radius:5px;
}
QGroupBox{
border:none;

}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>All Creditees</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_credits_new_creditee">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>New Creditee</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/user_add.png</normaloff>:/icons/user_add.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_all_creditees">
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(214, 214, 214);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>100</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Mobile Number : </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_credit_details_mobile">
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>60</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="viewcredititemsBtn">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(143, 69, 255);</string>
                  </property>
                  <property name="text">
                   <string>View All Items Taken On Credit</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/icons/user_edit.png</normaloff>:/icons/user_edit.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <property name="text">
               <string>                       Amount Paid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_credit_pay_amount">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Enter Amount paid in Ksh.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_credit_pay_update">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 170, 127);</string>
              </property>
              <property name="text">
               <string>Repay Debt</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(161, 161, 161);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Credits Taken</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_pay_debt_details_3">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>140</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Debt Balance</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="background">
                 <color>
                  <red>240</red>
                  <green>240</green>
                  <blue>240</blue>
                 </color>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Debt Amount</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="background">
                 <color>
                  <red>240</red>
                  <green>240</green>
                  <blue>240</blue>
                 </color>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Date Taken (Year-Month-Day)</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="background">
                 <color>
                  <red>240</red>
                  <green>240</green>
                  <blue>240</blue>
                 </color>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Due Date (Year-Month-Day)</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="background">
                 <color>
                  <red>240</red>
                  <green>240</green>
                  <blue>240</blue>
                 </color>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Payments Done</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_pay_debt_details_2">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>140</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Debt Balance</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Amount Paid</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Date Paid (Year-Month-Day)</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="foreground">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>70</red>
                   <green>70</green>
                   <blue>70</blue>
                  </color>
                 </brush>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_credits_back">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 170, 0);
color: rgb(0,0,0);</string>
              </property>
              <property name="text">
               <string>&amp; &lt;&lt; Back   </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
